data/data/fcos.json

```diff
@@ -1,166 +1,149 @@
+        "url": "https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/32.20200601.3.0/x86_64/fedora-coreos-32.20200601.3.0-azure.x86_64.vhd.xz"
+    },
+    "baseURI": "https://builds.coreos.fedoraproject.org/prod/streams/next/builds/32.20200601.3.0/x86_64/",
+    "buildid": "32.20200601.3.0",
     "gcp": {
-        "image": "fedora-coreos-32-20200517-1-0-gcp-x86-64",
-        "url": "https://storage.googleapis.com/fedora-coreos-cloud-image-uploads/image-import/fedora-coreos-32-20200517-1-0-gcp-x86-64.tar.gz"
+        "family": "fedora-coreos-stable",
+        "image": "fedora-coreos-32-20200601-3-0-gcp-x86-64",
+        "project": "fedora-coreos-cloud",
+        "url": "https://storage.googleapis.com/fedora-coreos-cloud-image-uploads/image-import/fedora-coreos-32-20200601-3-0-gcp-x86-64.tar.gz"
     },
     "images": {
         "aliyun": {
-            "path": "fedora-coreos-32.20200517.1.0-aliyun.x86_64.qcow2.xz",
-            "sha256": "00d463bb75ae98d723ae03ba50dc38a4790fd88dcf5767a2753b7758a70a9662",
-            "size": 503483176,
-            "uncompressed-sha256": "6e3a3b0646a46caead3eb00754e9d0757a35e4cc120cacad2bf6732f9bee7c51",
-            "uncompressed-size": 1774845952
+            "path": "fedora-coreos-32.20200601.3.0-aliyun.x86_64.qcow2.xz",
+            "sha256": "4b21be940efe60957adb1440712b702d1ad0f09ec598455555ced27bac6195e7",
+            "size": 503463040,
+            "uncompressed-sha256": "1aeab8b63ac957f1ba7b7de588fa329d4d201ed272413e8ca2c4d8f245358110",
+            "uncompressed-size": 1776746496
         },
         "aws": {
-            "path": "fedora-coreos-32.20200517.1.0-aws.x86_64.vmdk.xz",
-            "sha256": "ace6a2226cb9142cdcc41f00849bcbe45e761f9c9b339e7fa4a9ed79fadf363b",
-            "size": 734159676,
-            "uncompressed-sha256": "aa8e537f7fe5644ade9f58d67fddda705b9b99bbfe10f4f52d52906f0de34996",
-            "uncompressed-size": 775048192
+            "path": "fedora-coreos-32.20200601.3.0-aws.x86_64.vmdk.xz",
+            "sha256": "5683bad6307ea1626725c79fc88f38ba0ac32c847755b3b2f3e126ed47a66cd5",
+            "size": 734917852,
+            "uncompressed-sha256": "d209a6807936b306a1612261bfdd4f3653bd0a565a453cbc6296c267f9526f07",
+            "uncompressed-size": 775761408
         },
         "azure": {
-            "path": "fedora-coreos-32.20200517.1.0-azure.x86_64.vhd.xz",
-            "sha256": "fad529db18348874827d4ef8522ff36748dedca723355e4d60069045e39be3ab",
-            "size": 504146464,
-            "uncompressed-sha256": "6c5e1394fe62dee5af7dca9137b16226fb7aa09f6836cea66de025d9b9d932f8",
+            "path": "fedora-coreos-32.20200601.3.0-azure.x86_64.vhd.xz",
+            "sha256": "7d7abd74f32ee159e90bd23ebbbb882d9716cb176a0349af23876fc705820132",
+            "size": 503626396,
+            "uncompressed-sha256": "0990669f182eb1241716d0aa50207a38cf2551bd3ad6b154e5af57d1848e05d3",
             "uncompressed-size": 8589935104
         },
-        "digitalocean": {
```

Comment:
Looks like these got lost?

Reply:
These are not used by installer, so I have to remove these